### PR TITLE
Feat(Query):  Kubelet Client Periodic Certificate Switch Disabled

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Kubelet Client Periodic Certificate Switch Disabled

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/test/negative1.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/test/negative1.yaml:10]: This image is for `kube-apiserver`, you should use the kubelet docker image
```suggestion
      image: joaodanielrufino/kubelet
```

File: assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/test/positive1.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: command-demo
+  labels:
+    purpose: demonstrate-command
+spec:
+  containers:
+    - name: command-demo-container
+      image: gcr.io/google_containers/kube-apiserver-amd64:v1.6.0
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/test/positive1.yaml:10]: Same here

File: assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	command := "kubelet"
+
+	common_lib.inArray(container.command, command)
+	k8sLib.hasFlag(container, "--rotate-certificates=false")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("spec.command", []),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "--rotate-certificates flag is true",
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/query.rego:20]:
```suggestion
		"keyExpectedValue": "--rotate-certificates flag to be true",
```

File: assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/query.rego [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+CxPolicy[result] {
+	resource := input.document[i]
+    resource.kind == "KubeletConfiguration"
+    resource.rotateCertificates == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": "kind={{KubeletConfiguration}}.rotateCertificates",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "rotateCertificates is true",
```
Comment (joaorufi) [inline assets/queries/k8s/kubelet_client_periodic_certificate_switch_disabled/query.rego:35]:
```suggestion
		"keyExpectedValue": "rotateCertificates to be true",
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — DISMISSED** [review]
LGTM 🚀

**Review by rafaela-soares — APPROVED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**